torch/csrc/distributed/c10d/NCCLUtils.hpp

```diff
@@ -234,6 +234,11 @@ class NCCLComm {
       std::vector<uint64_t>& ranks_ull);
 #endif // NCCL_HAS_COMM_SPLIT
 
+#ifdef NCCL_SIM_INFO_INITIALIZER
+  static float ncclCollectiveEstimateEnd(
+      std::unique_ptr<ncclSimInfo_t> simInfo);
+#endif // NCCL_SIM_INFO_INITIALIZER
+
```

Comment:
We can remove the `#ifdef` if we remove `ncclSimInfo_t` from this signature. See my other comment in cpp.

Minor name pref: `endSimulate`